src/aws_encryption_sdk/__init__.py

```diff
@@ -185,6 +185,9 @@ def decrypt(self, **kwargs):
                 If source_length is not provided and read() is called, will attempt to seek()
                 to the end of the stream and tell() to find the length of source data.
 
+        :param dict encryption_context: Dictionary defining encryption context to validate
+            on decrypt. This is ONLY validated on decrypt if using the required encryption
+            context CMM from the aws-cryptographic-materialproviders library.
```

Comment:
This should be validated by the default CMM as well.

Reply:
Note: "default CMM" is ONLY the MPL default CMM. The native ESDK-Python default CMM will NOT validate this.
All native ESDK-Python CMMs will be marked as deprecated as part of the release for MPL integration.
Deprecated components will not support new MPL features (like reproduced encryption context on decrypt).
(We may add support for reproduced EC to native default CMM at some point, but not for MPL integration launch.)